### DISCUSSION TITLE
Search in files with no file extension

### DISCUSSION THREAD

**Discussion by NotMyFault** [community/community#12055] [Other Feature Feedback, Questions, & Ideas] [answered]
Heyo,

I noticed, that files without file extensions, for example "Jenkinsfile"s, are either not indexed at all or do not occur in search results.
I was playing around with the query `org:jenkinsci "buildPlugin"`, which should list several hundreds of plugins, but no results were found.
Am I doing something wrong, or are these kinds of files actually not indexed at the moment?

Kind regards

**Comment by colinwm** [comment, marked as answer]
Hey @NotMyFault , a number of files are currently being excluded because we detect them as vendored or generated, and I think these Jenkinsfiles fall into one of those categories. For more information, see https://cs.github.com/about/faq#indexed-content.

We understand that this limitation can be pretty confusing, and they often exclude code that people do want to search for. We're working on removing all of those restrictions. I'll give another update when we've removed them.
